kubert/src/client.rs

```diff
@@ -1,10 +1,11 @@
 //! Utilities for configuring a [`kube_client::Client`] from the command line
 
+use kube_client::config::Kubeconfig;
```

Comment:
If we're going to import this we should probably also update the use of `KubeConfigOptions` ([below also](https://github.com/olix0r/kubert/pull/21/files#diff-2f4fccfe3ccd43d4bbf2382abc5c9b11e64e7d6c607b5fd317bce53845ae5eecR64))

```suggestion
use kube_client::config::{Kubeconfig, KubeConfigOptions};
```